crates/persistence/src/error/mod.rs

```diff
@@ -0,0 +1,198 @@
+//! This module contains Error type definitions that are used throughout Holochain, and the Ribosome in particular,
+//! which is responsible for mounting and running instances of DNA, and executing WASM code.
+
+use self::PersistenceError::*;
+use crate::json::*;
+use futures::channel::oneshot::Canceled as FutureCanceled;
+use serde_json::Error as SerdeError;
+use std::{
+    error::Error,
+    fmt,
+    io::{self, Error as IoError},
+    option::NoneError,
+};
+
+//--------------------------------------------------------------------------------------------------
+// CoreError
+//--------------------------------------------------------------------------------------------------
```

Comment:
dead comment

Reply:
> dead comment

Removed in 28de48bc064286a0cae8694053e33048ad4326c0.